src/renderers/noop/ReactNoop.js

```diff
@@ -160,6 +172,14 @@ var ReactNoop = {
     }
   },
 
+  hasScheduledDeferredCallback() {
+    return Boolean(scheduledDeferredCallback);
```

Comment:
IMO ReactNoop served as a good example of a minimal renderer for documentation purposes but also because there is minimal logic in the renderer so it creates a nice unit of work.

The more logic the renderer has or exposes, the less we're testing only the renderer code and we might rely on things in the ReactNoop renderer that we forget to replicate in other renderers.